react/features/recording/components/LiveStream/native/StreamKeyPicker.js

```diff
@@ -67,10 +67,22 @@ class StreamKeyPicker extends Component<Props, State> {
     render() {
         const { broadcasts } = this.props;
 
-        if (!broadcasts || !broadcasts.length) {
+        if (!broadcasts) {
             return null;
         }
 
+        if (!broadcasts.length) {
+            return (
+                <View style = { styles.formWrapper }>
+                    <View style = { styles.streamKeyPickerCta }>
+                        <Text>
+                            { this.props.t('liveStreaming.noStreamFound') }
```

Comment:
I don't fully understand why there is no youtube link added to mobile if no keys found. Can you please explain a bit more?

Reply:
When I had tested the link, because it's a youtube link, I noticed it was first trying to open the youtube app by default as I had no preference set. When I would then select to open in chrome, I would be presented with a login screen. Compare this with web where the link will open a new tab and the user is already logged in due to logging in to use the chrome api. I was unsure this experience would be nice so I left it out of mobile.  Do you feel it's worth keeping it in versus leaving it out?
![linking](https://user-images.githubusercontent.com/1243084/50239109-1d0e2000-0376-11e9-8a78-34b3d90b0c41.gif)

Reply:
I think I convinced myself to put it back in.